app/components/admin/teacher-credentials-modal.tsx

```
@@ -46,16 +59,25 @@ export function TeacherCredentialsModal({
       <div className="bg-white rounded-lg shadow-xl max-w-md w-full p-6 m-4">
         <div className="mb-4">
           <h2 className="text-2xl font-bold text-green-600 mb-2">
-            ✓ Account Created Successfully
+            ✓ {title || defaultTitle}
           </h2>
-          <p className="text-gray-600">
-            Teacher account for <strong>{teacherName}</strong> has been created.
-          </p>
+          <p
+            className="text-gray-600"
+            dangerouslySetInnerHTML={{ __html: description || defaultDescription }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)